---
title: "Software & Resources"
---

![](stickers/Bioconductor.png){width=2.5cm}![](stickers/CATALYST.png){width=2.5cm}![](stickers/countsimQC.png){width=2.5cm}![](stickers/crispRVariants.png){width=2.5cm}![](stickers/DRIMSeq.png){width=2.5cm}![](stickers/edgeR.png){width=2.5cm}![](stickers/iCOBRA.png){width=2.5cm}![](stickers/regsplice.png){width=2.5cm}

### **R packages**

Name      | Author(s)               | Description 
----------|---------------|-----------------------------------
[CATALYST](http://bioconductor.org/packages/release/bioc/html/CATALYST.html) | **Helena L Crowell**<br>**Mark D Robinson**<br>Vito RT Zanotelli<br>St√©phane Chevrier<br>Bernd Bodenmiller | Tools for preprocessing of cytometry data including<br>normalization, debarcoding, and compensation
[countsimQC](https://github.com/csoneson/countsimQC) | **Charlotte Soneson** | Comparison of characteristic features of count data sets
[CrispRVariants](http://bioconductor.org/packages/release/bioc/html/CrispRVariants.html) | **Helen Lindsay** | Tools for counting and visualising mutations in a target location
[DRIMSeq](http://bioconductor.org/packages/release/bioc/html/DRIMSeq.html) | **Malgorzata Nowicka** | Differential transcript usage and tuQTL analyses<br>with Dirichlet-multinomial model in RNA-seq
[edgeR](http://bioconductor.org/packages/release/bioc/html/edgeR.html) | Yunshun Chen<br>Aaron Lun<br>Davis McCarthy<br>**Xiaobei Zhou**<br>**Mark D Robinson**<br>Gordon Smyth | Empirical Analysis of Digital Gene Expression Data in R
[flagme](http://bioconductor.org/packages/release/bioc/html/flagme.html) | **Mark D Robinson**<br>Riccardo Romoli | Analysis of Metabolomics GC/MS Data
[iCOBRA](http://bioconductor.org/packages/release/bioc/html/iCOBRA.html) | **Charlotte Soneson** | Comparison and Visualization of Ranking and Assignment Methods
[regsplice](http://bioconductor.org/packages/release/bioc/html/regsplice.html) | **Lukas M Weber** | L1-regularization based methods for detection of differential splicing
[Repitools](http://bioconductor.org/packages/release/bioc/html/Repitools.html) | **Mark D Robinson**<br>Dario Strbenac<br>Aaron Statham<br>Andrea Riebler | Epigenomic tools

### **Web applications**

Name      | Author(s)     | Description
----------|---------------|-----------------------------------
[CATALYST](https://catalyst-project.github.io) | **Helena L Crowell** | Tools for preprocessing of cytometry data including<br>normalization, debarcoding, and compensation
[conquer](http://imlspenticton.uzh.ch:3838/conquer/) | **Charlotte Soneson** | Repository providing consistently processed<br>public single-cell RNA-seq data sets
[CrispantCal](http://lmweber.github.io/CrispantCal/) | **Lukas M Weber** | R/Shiny web app to calculate injection mix volumes<br>for CRISPR-Cas9 experiments
[CrispRVariantsLite](http://imlspenticton.uzh.ch:3838/CrispRVariantsLite/) | **Helen Lindsay** | Graphical user interface to the R/Bioconductor CrispRVariants package